internal/pkg/mapper/orchestrator/parse_task.go

```diff
@@ -45,6 +45,20 @@ func (p *taskParser) name() (string, error) {
 	return value, nil
 }
 
+func (p *taskParser) enabled() (bool, error) {
+	raw, found := p.content.Get(`enabled`)
+	if !found {
+		// Use default value
+		return true, nil
+	}
+	value, ok := raw.(bool) // JSON int is float64, by default in Go
```

Comment:
Můžeš tenhle komentář víc rozvést? Chápu, co to znamená, ale nevím, jak to souvisí s tím kódem

Reply:
Je to jednoduche :slightly_smiling_face:  zabudol som zmazat komentar, ked som skopiroval kod a nevsimol som si to (v inej casti kodu je `.(float64)`).

Dik :+1:  , opravene tu https://github.com/keboola/keboola-as-code/pull/817/commits/fca0f1cf0beccdf97130774441c424ade6a13fba